lang/ilias_de.lang

```diff
@@ -2944,6 +2944,7 @@ common#:#info_deactivate_sure#:#Sind Sie sicher, dass Sie die folgenden Benutzer
 common#:#info_delete_sure#:#Sind Sie sicher, dass folgende Objekte gelöscht werden sollen:
 common#:#info_remove_sure#:#Sind Sie sicher, dass folgende Objekte entfernt werden sollen:
 common#:#info_delete_warning_no_trash#:#(WARNUNG: Die ausgewählten Objekte werden unwiderruflich aus dem System entfernt und können nicht wiederhergestellt werden!)
+common#:#info_delete_warning_no_trash_for_org_units#:# (WARNUNG: Organisationseinheiten werden unwiderruflich aus dem System entfernt und können nicht wiederhergestellt werden!)
```

Comment:
Thanks, please add the english translation as well

Reply:
@chfsx The english translation exists on my local bug fix branch.